vendor/vendor.json

```diff
@@ -21,5 +21,5 @@
 			"revisionTime": "2017-05-08T18:43:26Z"
 		}
 	],
-	"rootPath": "github.com/jaypipes/ghw/ghwc"
+	"rootPath": "github.com/jaypipes/ghw"
```

Comment:
So, the reason I had the ghwc/vendor folder was that I did not want to make cobra a dependency for the ghw library itself. I wanted to have as few (or no) dependencies for the importable library as possible. Is there a way to indicate that the dependency is only for the ghwc example client tool and not the library itself?

Reply:
Ah, I seee.  Now I understand what ghwc was for.